plugins/connection/logix.py

```diff
@@ -112,7 +112,6 @@ def __init__(self, play_context, new_stdin, *args, **kwargs):
             )
 
         self.host = self.get_option("host")
-        self._sub_plugin = {"type": "external"}
```

Comment:
I think we need to leave this line in, otherwise everything else looks good.

Reply:
You really don't. As I said before, the whole point of PersistentConnectionBase was to keep you from needing to have that useless arcane line.

Additionally, because task_executor makes certain assumptions about what the plugin looks like when it sees that attribute is set, you are likely to run into more trouble by having it than not, particularly if you want to set your own options on the connection.

Reply:
alright +1